build/_k-standard-goals.shade

```diff
@@ -95,10 +95,16 @@ default Configuration='${E("Configuration")}'
         var msbuildPath =  Path.Combine(programFilesX86, "MSBuild", msbuildVersions[i], "Bin", "MSBuild.exe");
         if (File.Exists(msbuildPath))
         {
+          var commonParameters =
+            " /p:Configuration=" + Configuration +
+            " /p:ProductVersion=1.0.0" +
```

Comment:
Newbie question - where can I get the product version from? Some repos (dnx) define `PRODUCT_VERSION` in the shade file. Some repos define `VERSION` but it does not seem to be used (UserSecrets define `VERSION=0.1` but the dll is `1.0.0....`). At the moment I hardcoded it but there should be only one place when we define this version.